### PR TITLE
port rqt_joint_trajectory_controller to ros2

### PR DESCRIPTION
**Pull request by Noel215**
Port over the package rqt_joint_trajectory_controller from https://github.com/ros-controls/ros_controllers/tree/noetic-devel/rqt_joint_trajectory_controller, which implements the rqt GUI to control the joint_trajectory_controller instances

PR for the following issue [#313](https://github.com/ros-controls/ros2_controllers/issues/313)

### CODE REVIEW

File: rqt_joint_trajectory_controller/plugin.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <statustip>Plugins related to robot tools.</statustip>
+      </group>
+      <label>Joint trajectory controller</label>
+      <icon>resource/scope.png</icon>
+      <statustip>
+        Monitor and send commands to running joint trajectory controllers.
+      </statustip>
+    </qtgui>
+  </class>
+</library>
```
Comment (homalozoa) [inline rqt_joint_trajectory_controller/plugin.xml:21]: Add new line here, please.

File: rqt_joint_trajectory_controller/setup.cfg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[develop]
+script-dir=$base/lib/rqt_joint_trajectory_controller
+[install]
+install-scripts=$base/lib/rqt_joint_trajectory_controller
```
Comment (homalozoa) [inline rqt_joint_trajectory_controller/setup.cfg:4]: New line here, too.

**Review by homalozoa — CHANGES_REQUESTED** [review]

**Review by homalozoa — APPROVED** [review]

**Review by bmagyar — CHANGES_REQUESTED** [review]

**Comment by Noel215** [comment]
The requested changes were pushed. I've been checking the failures in the workflows but all of them are related to other packages so I don't know how to continue with this PR

**Review by bmagyar — APPROVED** [review]

**Comment by bmagyar** [comment]
Sorry for the wait, I did a bit of simulation testing and will have a couple of follow-up tasks but this is good to go as-is! Thanks for sticking around

**Pull request merged**